App/Services/UserNotifier.cs

```diff
@@ -61,10 +71,18 @@ public Task ShowErrorNotification(string title, string message, CancellationToke
         return Task.CompletedTask;
     }
 
-    public Task ShowActionNotification(string title, string message, string handlerName, IDictionary<string, string>? args = null, CancellationToken ct = default)
+    public Task ShowActionNotification(string title, string message, string? handlerName, IDictionary<string, string>? args = null, CancellationToken ct = default)
     {
-        if (!Handlers.TryGetValue(handlerName, out _))
-            throw new InvalidOperationException($"No action handler with the name '{handlerName}' is registered.");
+        if (handlerName == null)
+        {
+            // Use default handler if no handler name is provided
+            handlerName = nameof(DefaultNotificationHandler);
+        }
+        else
+        {
+            if (!Handlers.TryGetValue(handlerName, out _))
+                throw new InvalidOperationException($"No action handler with the name '{handlerName}' is registered. Use null for default");
```

Comment:
This should be a comment on the interface method instead of in the exception description